site/index.html

```diff
@@ -547,6 +577,8 @@ <h2 class="white"><span class="lines">Contact</span></h2>
   <script src="javascripts/smooth-scroll.js" type="text/javascript"></script>
```

Comment:
We should use minimized versions for all these javascripts. Preferably from a CDN too if possible.